NEWS.md

```diff
@@ -1,3 +1,19 @@
+### v0.1.5
+
+Contributed by [@peter1000](https://github.com/peter1000)
+
+* Add additional output filtering based on category.
+* Add optional output grouping by category.
+* Add optional API-Index pages for markdown output.
+* Add optional Permalink for markdown output.
+* Adjusted some documentation for the new API-Index. (Shorter first line ect.)
+* Code reduction / simplification.
+* Trimmed most code lines to 100 chars where possible.
+* Adds `examples` older with
+    
+    * join_multiple_modules
+    * fixheader_css_mkdocs
```

Comment:
News items should be, for the most part, user-facing changes. Things like "code reduction..." and "trimmed most code..." don't need to feature on this page, since someone can go back through the git history if they really want to.